src/mokka/src/integration-test/java/pl/hycom/mokka/service/openapi/DefaultOpenApiParsingServiceIT.java

```diff
@@ -0,0 +1,65 @@
+package pl.hycom.mokka.service.openapi;
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import pl.hycom.mokka.Application;
+
+import java.io.*;
+import java.util.List;
+
+@DirtiesContext
+@ActiveProfiles("test")
+@AutoConfigureTestDatabase
+@SpringBootTest(classes = Application.class)
```

Comment:
Do we need to configure database and complete SpringBoot context here? Maybe we it would be enough to limit just to 
`@SpringBootTest(classes = ConversionConfiguration.class)`?

So then you could remove: AutoConfigureTestDatabase and DirtiesContext